dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerArguments.java

```diff
@@ -251,4 +259,15 @@ Path getExclusionFile() {
     }
     return null;
   }
+
+  /**
+   * Returns the path to write linkage errors as exclusion file. If the argument is not specified,
+   * {@code null}.
+   */
+  Path getWriteAsExclusionFile() {
```

Comment:
This should be renamed similarly. Maybe we can have inputExclusionFile and outputExclusionFile? or just outputExclusions?